sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -142,7 +159,10 @@ internal AnalyzeHealthcareEntitiesOperation(TextAnalyticsRestClient serviceClien
 
             // TODO: Add validation here
             // https://github.com/Azure/azure-sdk-for-net/issues/11505
-            Id = operationLocation.Split('/').Last();
+            _jobId = operationLocation.Split('/').Last();
+
+            string plainId = JsonSerializer.Serialize(new OperationIdInformation(_jobId, idToIndexMap, showStats));
```

Comment:
I know this was before we began to expose actionName, so at this time we only allowed an action per type. I can't remember with the latest version of .NET TA if you also need the action order in the operationID to rebuild the response.

Reply:
we don't use the action order in our beta's either. I believe only Python is taking that into consideration